Sources/App/Features/Past Sponsors/Models/Migrations/PastSponsorMigration.swift

```diff
@@ -0,0 +1,17 @@
+import Fluent
+
+class PastSponsorMigration: AsyncMigration {
```

Comment:
What is unique about PastSponsors that we can't achieve with a query against the main Sponsors table?

The sponsors entity already includes an "event_id" column, is past sponsors not simply "select all from sponsors where event_id is not the latest event" - a query which we can quite easily achieve?

Reply:
Yeah, so like our discussion on Slack - I would prefer to "pick and choose" what we display here

Reply:
I'd still prefer that be a "visible" flag on the database, though. It feels like quite a lot of effort to manage two representations of the same data just to differentiate when and where one is shown!